### ISSUE TITLE
Gaiacli version error makes Voyager crash

### ISSUE THREAD

**Issue by fedekunze** [cosmos/voyager#1169]
UI Version: 0.X.X

Description:

I had the v0.24.0 installed of the SDK and I got an error when running `yarn start`. Also `yarn build:gaia` wasn't very helpful.

```
electron  gaiacli version: "0.24.0", expected: "0.23.1"

electron  [Unhandled Promise Rejection] Error: Requires gaia 0.23.1, but got 0.24.0.
                Please update your gaiacli installation or build with a newer binary.
              at _callee12$ (/Users/federico/Projects/voyager/app/src/main/index.js:654:11)
              at tryCatch (/Users/federico/Projects/voyager/node_modules/regenerator-runtime/runtime.js:62:40)
              at Generator.invoke [as _invoke] (/Users/federico/Projects/voyager/node_modules/regenerator-runtime/runtime.js:296:22)
              at Generator.prototype.(anonymous function) [as next] (/Users/federico/Projects/voyager/node_modules/regenerator-runtime/runtime.js:114:21)
              at step (/Users/federico/Projects/voyager/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
              at /Users/federico/Projects/voyager/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:13
              at <anonymous>
              at process._tickDomainCallback (internal/process/next_tick.js:228:7)
```

**Comment by faboweb** [comment]
Which network are you trying to connect to? Probably gaia-7005 which expects `0.23.1`. Maybe delete the config folder of that network under `~/.voyager-dev/`.

**Comment by fedekunze** [comment]
```
federicos-mbp-2:voyager federico$ gaiacli version
0.23.1-23e3d5ac
```
Still happening

**Comment by fedekunze** [comment]
It's probably because the 7005 is down

**Comment by faboweb** [comment]
The Voyager Gaia binaries are in `{voyager-dir}/builds/gaia/darwin.../` try getting the version from that binary.

**Comment by fedekunze** [comment]
The main issue that I had with this bug is that it didn't allow me to change to the local testnet setting. I had to quit Voyager and run the CLI with `yarn build:local`

**Comment by fedekunze** [comment]
@faboweb : So `gaiacli version: "0.24.0", expected: "0.23.1"` has nothing to do with remote nodes. This just means `gaiacli version` is different then the content of the `gaiaversion.txt` that lives in `~/.voyager-dev/{NETWORK}`

**Comment by fedekunze** [comment]
Closed via #1187
